oe_paragraphs.post_update.php

```diff
@@ -246,3 +246,17 @@ function oe_paragraphs_post_update_10009(): void {
   $field_storage->set('settings', $settings);
   $field_storage->save();
 }
+
+/**
+ * Enable 'Description' field for 'Listing item' thumbnail primary variant.
+ */
+function oe_paragraphs_post_update_10010(): void {
```

Comment:
Let's not do the post update here since it will put the field on for all the sites and it won't do anything for them. Let's move it to ewcms pls.